src/viz-parole-population/VizParolePopulation.js

```diff
@@ -1,26 +1,129 @@
 import PropTypes from "prop-types";
 import React from "react";
+import { group, sum } from "d3-array";
 import styled from "styled-components";
+import {
+  DIMENSION_DATA_KEYS,
+  DIMENSION_KEYS,
+  DIMENSION_LABELS,
+  DIMENSION_MAPPINGS,
+  THEME,
+} from "../constants";
+import {
+  formatAsNumber,
+  formatDemographicValue,
+  recordIsTotal,
+} from "../utils";
+import ProportionalBar from "../proportional-bar";
 import StateDistrictMap from "../state-district-map";
-import { recordIsTotal } from "../utils";
+import Statistic from "../statistic";
+
+const BAR_CHART_VISUALIZATION_COLORS = {
+  [DIMENSION_KEYS.gender]: THEME.colors.gender,
+  [DIMENSION_KEYS.age]: THEME.colors.age,
+  [DIMENSION_KEYS.race]: THEME.colors.race,
+};
 
 const VizParolePopulationContainer = styled.div`
   display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
 `;
+
 const VizWrapper = styled.div`
-  flex: 0 1 auto;
+  flex: 0 1 49%;
```

Comment:
This section of the page is perhaps slightly unique in that it has multiple visualizations (the Map and the Statistic Number/Bar Charts).  I wanted to give equal width sizings (i.e. 50%) to each of these visualization sections, but when I did the visualization section on the right side wrapped.

Reply:
this seems fine to me ... i am doing another pass at the responsive layout for this section so if there is any new weirdness that crops up from that I will deal with it